joomla/joomla-cms
# Please allow sorting of articles by modification date #10355

Pull Request for Issue #10355
#### Summary of Changes

No column with modification date was present and sorting articles by the last modification date was not possible.
So I have added one column with the modified date and sorting articles by the last modification date is possible now.
#### Testing Instructions
1. Log in to the Administrator component
2. Go to Content->Articles
3. You will see new column i.e modified date. and you can sort articles by modified date.
